src/test_driver/linux-cirque/InteractionModelTest.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+#
+# Copyright (c) 2020 Project CHIP Authors
```

Comment:
might be 2021 by the time this lands :-)

Reply:
Done, thanks!